cmd/catalog.go

```diff
@@ -50,14 +50,32 @@ var catalogListComponentCmd = &cobra.Command{
 		case 0:
 			fmt.Printf("No deployable components found\n")
 		default:
-
+			currentProject := client.GetCurrentProjectName()
 			w := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
-			fmt.Fprintln(w, "NAME", "\t", "TAGS")
+			fmt.Fprintln(w, "NAME", "\t", "PROJECT", "\t", "TAGS")
 			for _, component := range catalogList {
-				fmt.Fprintln(w, component.Name, "\t", strings.Join(component.Tags, ","))
+				componentName := component.Name
+				if component.Namespace == currentProject {
+					/*
+						If current namespace is same as the current component namespace,
+						Loop through every other component,
+						If there exists a component with same name but in different namespaces,
+						mark the one from current namespace with (*)
+					*/
+					for _, comp := range catalogList {
+						if comp.Name == component.Name && component.Namespace != comp.Namespace {
+							componentName = fmt.Sprintf("%s (*)", component.Name)
+						}
+					}
+				}
+				fmt.Fprintln(w, componentName, "\t", component.Namespace, "\t", strings.Join(component.Tags, ","))
 			}
 			w.Flush()
-
+			fmt.Println(
+				`NOTE: There might be items in multiple projects.
```

Comment:
Honestly, this looks a bit ugly / out of place:

```sh
github.com/redhat-developer/odo  pr_618 ✔                                                                                                                                                                                                                                                                                                                            9h24m  
▶ ./odo catalog list components
NAME        PROJECT       TAGS                        
dotnet      openshift     2.0,latest
httpd       openshift     2.4,latest
nginx       openshift     1.10,1.12,1.8,latest                                                                                                                                                                                                                                                                                                                            
nodejs      openshift     0.10,4,6,8,latest
perl        openshift     5.16,5.20,5.24,latest
php         openshift     5.5,5.6,7.0,7.1,latest
python      openshift     2.7,3.3,3.4,3.5,3.6,latest                                                                                                                                                                                                                                                                                                                        
ruby        openshift     2.0,2.2,2.3,2.4,2.5,latest
wildfly     openshift     10.0,10.1,11.0,12.0,8.1,9.0,latest
NOTE: There might be items in multiple projects.
Items from the current project (marked with *) will take precedence.                                                                                                                                                                                                                                                                                                        
If you want to use an item from a different namespace, use the full name as <namespace>/<name>
```

I think that we should remove this note completely.

Reply:
Anyway removing it since we already also have it in our docs